### PR TITLE
Re-adds apache-rat. Cures missing license headers.

### PR DESCRIPTION
**Pull request by rzo1**
Thank you for contributing to Apache OpenNLP.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [ ] Does your PR title start with OPENNLP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically main)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via `mvn clean install` at the root opennlp-sandbox folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file in opennlp-sandbox folder?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found in opennlp-sandbox folder?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: opennlp-similarity/src/main/java/opennlp/tools/similarity/apps/SentenceTranslate.java.txt [hunk 1]
```diff
@@ -208,5 +227,5 @@ public class SentenceTranslate {
          * @param args
          */
         public static void main(String[] args) throws Exception {
-                SentenceTranslate.improveSentenceTranslationBySimilartyAssessment(SentenceTranslate.executeByMicrosoftTranslator("� �����������, 8 �������, ������ ����� ����� �������� � ���������� ����� ���� ���������������� �������, ������� � ���� ���� ����� �������."));
+                SentenceTranslate.improveSentenceTranslationBySimilartyAssessment(SentenceTranslate.executeByMicrosoftTranslator("� �����������, 8 �������, ������ ����� ����� �������� � ���������� ����� ���� ���������������� �������, ������� � ���� ���� ����� �������."));
```
Comment (rzo1) [inline opennlp-similarity/src/main/java/opennlp/tools/similarity/apps/SentenceTranslate.java.txt:230]: No changes from my side. Seems file was not correctly encoded before (utf-8) and is broken anyway.

**Review by kinow — APPROVED** [review]

**Comment by kinow** [comment]
@rzo1 

>Error:    GradientDescentUtilsTest.testConvergence:34 » Runtime failed to converge at it...

Is that a brittle test? I seem to recall it being mentioned as failing.. but I thought it was on macos?

**Comment by rzo1** [comment]
Yes. Flaky test ;)

File: opennlp-similarity/src/main/java/opennlp/tools/similarity/apps/SentenceTranslate.java.txt (hunk 1 above)
Comment (mawiesne) [reply opennlp-similarity/src/main/java/opennlp/tools/similarity/apps/SentenceTranslate.java.txt:230]: Strange it ends with `.txt`...

**Comment by mawiesne** [comment]
> Yes. Flaky test ;)

@kinow @rzo1 I've opened a Jira for this. See: https://issues.apache.org/jira/browse/OPENNLP-1472

File: opennlp-similarity/src/main/java/opennlp/tools/similarity/apps/SentenceTranslate.java.txt (hunk 1 above)
Comment (rzo1) [reply opennlp-similarity/src/main/java/opennlp/tools/similarity/apps/SentenceTranslate.java.txt:230]: Indeed.

**Review by mawiesne — APPROVED** [review]

**Pull request merged**